canadian-journal-of-earth-science.csl

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Canadian Journal of Earth Sciences</title>
+    <id>http://www.zotero.org/styles/canadian-journal-of-earth-science</id>
```

Comment:
@libora, this also needs to be "sciences", not "science". (http://www.nrcresearchpress.com/journal/cjes)